### PR TITLE
fix: add solhint files and docs

### PR DESCRIPTION
**Pull request by rin-st**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -149,6 +149,8 @@ yarn hardhat verify --constructor-args arguments.js DEPLOYED_CONTRACT_ADDRESS
 
 # Linting
 
+Install solhint globally [Docs](https://protofire.github.io/solhint/#installation)
```
Comment (krakxn) [inline README.md:152]:
```suggestion
`solhint` installation: [Documentation](https://protofire.github.io/solhint/#installation)
```

More apt.

File: package.json [hunk 2]
```diff
@@ -18,7 +18,7 @@
   "scripts": {
     "test": "hardhat test",
     "test:staging": "hardhat test --network goerli",
-    "lint": "solhint 'contracts/*.sol'",
+    "lint": "solhint 'contracts/**/*.sol'",
```
Comment (krakxn) [inline package.json:21]: This change seems unnecessary, as providing the base path--i.e. `contracts/*`--is sufficient.

Please correct me if I am overlooking this.

**Review by krakxn — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (rin-st) [reply package.json:21]: see next line. I believe paths should be equal. Also `contracts/*` doesn't count `test` folder.

**Comment by PatrickAlphaC** [comment]
This is great! Thankyou!

**Pull request merged**